controllers/serviceinstance_controller.go

```diff
@@ -554,6 +554,11 @@ func updateRequired(serviceInstance *servicesv1.ServiceInstance) bool {
 		return false
 	}
 
+	cond := meta.FindStatusCondition(serviceInstance.Status.Conditions, api.ConditionSucceeded)
```

Comment:
This was a bug, in case update failed and it was in state: updateInProgress this function used to return false instead of true